ui-v2/app/components/phrase-editor.js

```diff
@@ -1,44 +1,67 @@
 import Component from '@ember/component';
 import { get, set } from '@ember/object';
+import { inject as service } from '@ember/service';
 
 export default Component.extend({
+  dom: service('dom'),
   classNames: ['phrase-editor'],
   item: '',
-  remove: function(index, e) {
-    this.items.removeAt(index, 1);
-    this.onchange(e);
+  didInsertElement: function() {
+    this._super(...arguments);
+    // TODO: ref
```

Comment:
What does this TODO mean?

Reply:
It's a reference to `{{ref}}`

https://octane-guides-preview.emberjs.com/release/components/glimmer-components-dom/#toc_communicating-between-elements-in-a-component

I added a little more detail to the comment in another commit

Reply:
ooOOOoo, so Reacty. I'm looking forward to that.